exchanges/bybit/bybit_test.go

```diff
@@ -85,3423 +78,3140 @@ func TestGetKlines(t *testing.T) {
 	if err != nil {
 		t.Error(err)
 	}
-}
-
-func TestGetAccountRatio(t *testing.T) {
-	t.Parallel()
-	pair, err := currency.NewPairFromString("BTCUSD")
+	_, err = b.GetOrderInfo(context.Background(),
+		"12234", inverseTradablePair, asset.CoinMarginedFutures)
 	if err != nil {
-		t.Fatal(err)
+		t.Error(err)
 	}
-
-	_, err = b.GetAccountRatio(context.Background(), pair, "1d", 0)
+	_, err = b.GetOrderInfo(context.Background(),
+		"12234", optionsTradablePair, asset.Options)
 	if err != nil {
 		t.Error(err)
 	}
 }
 
-func TestGetRiskLimit(t *testing.T) {
+func TestGetActiveOrders(t *testing.T) {
 	t.Parallel()
-	pair, err := currency.NewPairFromString("BTCUSD")
-	if err != nil {
-		t.Fatal(err)
+	if mockTests {
+		t.Skip("skipping authenticated function for mock testing")
 	}
-
-	_, err = b.GetRiskLimit(context.Background(), pair)
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
+	var getOrdersRequestSpot = order.MultiOrderRequest{
+		Pairs:     currency.Pairs{spotTradablePair},
+		AssetType: asset.Spot,
+		Side:      order.AnySide,
+		Type:      order.AnyType,
+	}
+	_, err := b.GetActiveOrders(context.Background(), &getOrdersRequestSpot)
 	if err != nil {
 		t.Error(err)
 	}
-}
-
-func TestGetLastFundingRate(t *testing.T) {
-	t.Parallel()
-	pair, err := currency.NewPairFromString("BTCUSD")
+	var getOrdersRequestLinear = order.MultiOrderRequest{Pairs: currency.Pairs{usdtMarginedTradablePair}, AssetType: asset.USDTMarginedFutures, Side: order.AnySide, Type: order.AnyType}
+	_, err = b.GetActiveOrders(context.Background(), &getOrdersRequestLinear)
 	if err != nil {
-		t.Fatal(err)
+		t.Error(err)
 	}
-
-	_, err = b.GetLastFundingRate(context.Background(), pair)
+	var getOrdersRequestInverse = order.MultiOrderRequest{Pairs: currency.Pairs{inverseTradablePair}, AssetType: asset.CoinMarginedFutures, Side: order.AnySide, Type: order.AnyType}
+	_, err = b.GetActiveOrders(context.Background(), &getOrdersRequestInverse)
 	if err != nil {
 		t.Error(err)
 	}
-}
-
-func TestGetFutureServerTime(t *testing.T) {
-	t.Parallel()
-	_, err := b.GetFuturesServerTime(context.Background())
+	var getOrdersRequestFutures = order.MultiOrderRequest{Pairs: currency.Pairs{optionsTradablePair}, AssetType: asset.Options, Side: order.AnySide, Type: order.AnyType}
+	_, err = b.GetActiveOrders(context.Background(), &getOrdersRequestFutures)
 	if err != nil {
 		t.Error(err)
 	}
 }
 
-func TestGetAnnouncement(t *testing.T) {
+func TestGetOrderHistory(t *testing.T) {
 	t.Parallel()
-	_, err := b.GetAnnouncement(context.Background())
+	if mockTests {
+		t.Skip("skipping authenticated function for mock testing")
+	}
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
+	var getOrdersRequestSpot = order.MultiOrderRequest{
+		Pairs:     currency.Pairs{spotTradablePair},
+		AssetType: asset.Spot,
+		Type:      order.AnyType,
+		Side:      order.AnySide,
+	}
+	_, err := b.GetOrderHistory(context.Background(), &getOrdersRequestSpot)
```

Comment:
Order status not being handled correctly:

raw:
```
{"orderLinkId":"16986271938551036","orderId":"1542591139945660160","blockTradeId":"","symbol":"BTCUSDT","price":"1000.00","isLeverage":"0","positionIdx":0,"qty":"0.005000","side":"Buy","ord
erStatus":"Cancelled","cancelType":"UNKNOWN","rejectReason":"","avgPrice":"0","leavesQty":"","leavesValue":"","cumExecQty":"0","cumExecValue":"","cumExecFee":"","timeInForce":"GTC","orderType":"Limit","stopOrderType":"","orderIv":"","triggerPrice":"","takeProfit":"","stopLoss":"","tpTriggerBy":"","slTriggerBy":"","triggerDirection":0,"triggerBy":"","lastPriceOnCreated":"","reduceOnly":false,"closeOnTrigger":false,"createdTime":"1698627193860","updatedTime":"1698627345185","smpType":"None","smpGroup":0,"smpOrderId":""}
```

Return:
```
{ImmediateOrCancel:false HiddenOrder:false FillOrKill:false PostOnly:false ReduceOnly:false Leverage:0 Price:1000 Amount:0.005 ContractAmount:0 LimitPriceUpper:0 LimitPriceLower:0 TriggerPrice:0 AverageExecutedPrice:0 QuoteAmount:0 ExecutedAmount:0 RemainingAmount:0 Cost:0 CostAsset:USDT Fee:0 FeeAsset: Exchange:Bybit InternalOrderID
:00000000-0000-0000-0000-000000000000 OrderID:1542591139945660160 ClientOrderID:16986271938551036 AccountID: ClientID: WalletAddress: Type:LIMIT Side:BUY Status:UNKNOWN AssetType:spot Date:2023-10-30 11:53:13.86 +1100 AEDT CloseTime:0001-01-01 00:00:00 +0000 UTC LastUpdated:2023-10-30 11:55:45.185 +1100 AEDT Pair:BTC_USDT MarginType:unset Trades:[]}
```

Reply:
All options elements are returning an empty list. With a "SUCCESS" message and that was the source of Unmarshal error. Therefore, I have added a minor convert code to act on such cases.

Reply:
> Status confirmed 👍